.orchestra/config/components/source-highlight.yml

```diff
@@ -0,0 +1,46 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+#@ load("/lib/make.lib.yml", "make")
+
+#@ source_url = "https://ftp.gnu.org/gnu/src-highlite/source-highlight-3.1.9.tar.gz"
+
+#@yaml/text-templated-strings
+---
+#@ def source_highlight_args():
+license: COPYING
+configure: |
+  mkdir -p "$BUILD_DIR"
+  extract.sh --into "$BUILD_DIR" (@= source_url @)
+  
+  cd "$BUILD_DIR"
+  sed 's/ throw [(]IOException[)]//g' -i lib/srchilite/fileutil.h lib/srchilite/fileutil.cc
+  
+  CC=clang CXX=clang++ CXXFLAGS=-stdlib=libc++ ./configure \
```

Comment:
Drop `CXXFLAGS=-stdlib=libc++`, our magic compiler wrapper will provide it silently.
```
.orchestra/config/global_options.yml:hard_flags_cxx_clang: -stdlib=libc++
```